### PR TITLE
Dockerized, added logging, fixed serialization

### PR DESCRIPTION
**Pull request by gznznzjsn**

### CODE REVIEW

File: .github/workflows/image-push.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Build and Push to Docker Hub
+
+on:
+  push:
+    branches: [ "dev" ]
```
Comment (h1alexbel) [inline .github/workflows/image-push.yml:5]: it's better to run workflows on push and pull-requests in `master` branch

**Review by h1alexbel — CHANGES_REQUESTED** [review]
@gznznzjsn 
decompose this pr into k8s-related changes and changes related to Serializers and other application logic

**Comment by gznznzjsn** [comment]
@h1alexbel everything is done

File: src/main/java/com/gznznzjsn/taskservice/web/kafka/RequirementSender.java [hunk 2]
```diff
@@ -17,6 +19,7 @@ public class RequirementSender {
     private final KafkaSender<String, Requirement> sender;
 
     public void send(Requirement requirement) {
+        log.info("Started sending of {}", requirement);
         this.sender.send(
                 Mono.just(
                         SenderRecord.create(
```
Comment (h1alexbel) [inline src/main/java/com/gznznzjsn/taskservice/web/kafka/RequirementSender.java:25]: it's not good to hardcode the message key. Messages with the same key will be sent in one partition

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/java/com/gznznzjsn/taskservice/web/kafka/RequirementSender.java (hunk 2 above)
Comment (gznznzjsn) [reply src/main/java/com/gznznzjsn/taskservice/web/kafka/RequirementSender.java:22]: Done

**Review by h1alexbel — APPROVED** [review]
OK

**Pull request merged**